### DISCUSSION TITLE
Openssl .cnf format and usage

### DISCUSSION THREAD

**Discussion by D3850** [openssl/openssl#25909] [Q&A] [answered]
Good morning. I am trying to create/use a .cnf file when creating a CSR and trying to locate the proper format and cli usage.
I an running openssl from a Win11 OS, and any help would be greatly appreciated.

Thank you in advance

Don

**Comment by t8m** [comment, marked as answer]
This is a good starting point for reading: https://docs.openssl.org/master/man5/config/

**Reply by D3850** [reply]
Thank you!
